data_type.js

```diff
@@ -0,0 +1,99 @@
+let challenge =  '30 Days Of JavaScript'
+let companies = 'Facebook, Google, Microsoft, Apple, IBM, Oracle, Amazon'
+let string = 'You cannot end a sentence with because because because is a conjunction'
+let challengeWithSpace = ' 30 Days Of JavaScript '
+let patternA = /a/gi  
+let challengePart1 =  '30 Days Of'
+let challengePart2 = 'JavaScript'
+
+console.log("2.Print the string: "+ challenge) 
+console.log("3.Print the length of the string: " + challenge.length)
+console.log("4.Change all the string characters to capital letters: "+ challenge.toUpperCase()) 
+console.log("5.Change all the string characters to lowercase letters: " + challenge.toLowerCase()) 
+console.log("6.Cut (slice) out the first word of the string: " + challenge.substring(0,2)) 
+console.log("7.Slice out the phrase: " + challenge.substring(3,21)) 
+console.log("8.Check if the string contains a word Script: "+ challenge.includes("Script"))
+console.log(challenge.split())
+console.log(challenge.split(" "))
+
+console.log(companies.split(',')) 
+
+console.log(challenge.replace('JavaScript', 'Python')) 
+console.log(challenge.charAt(15))  
+console.log(challenge.charCodeAt(challenge.indexOf('JavaScript'))) 
+console.log(challenge.indexOf('a'))  
+console.log(challenge.lastIndexOf('a'))  
+
+console.log(string.indexOf('because'))
+console.log(string.lastIndexOf('because'))
+console.log(string.search('because'))  
+
+console.log(challengeWithSpace)
+console.log(challengeWithSpace.trim(' '))
+
+console.log(challenge.startsWith('30')) 
+console.log(challenge.endsWith('Script')) 
+
+
+console.log(challenge.match(patternA)) 
+console.log(challengePart1.concat(" ",challengePart2))
+console.log(challenge.repeat(2))
+
+
+
+//level 2
+
+console.log('%cThe quote '+'%c\'There is no exercise better for the heart than reaching down and lifting people up.\' '+ '%cby John Holmes teaches us to '+ '%chelp '+ '%cone another.', 'color:#1F2328','color:#0A3069', 'color:#1F2328', 'color:#0550AE', 'color:#1F2328')
+
+console.log('%c\"Love is not patronizing and charity isn\'t about pity, it is about love. Charity and love are the same -- with charity you give love, so don\'t just give money but reach out your hand instead.\"', 'color:#0A3069')
+
+//Check if typeof '10' is exactly equal to 10. If not make it exactly equal.
+console.log(typeof('10')==typeof(10))
```

Comment:
compare `"10"` and `10` with strict equality operator. In real life, a text of `"10"` can not be equal to a number of `10`.